JS/fluentRevealNavbar.uc.js

```diff
@@ -135,10 +184,17 @@
       let { gradientSize, lightColor } = this._options;
       let isBookmark =
         el.id === "PlacesChevron" || el.classList.contains("bookmark-item");
-      let area = isBookmark
+      let area = isBookmark 
         ? el
-        : el.querySelector(".toolbarbutton-badge-stack") ||
-          el.querySelector(".toolbarbutton-icon");
+         : el.querySelector(".toolbarbutton-badge-stack") ||
+           el.querySelector(".toolbarbutton-icon");
+      if (!area && el.id == "urlbar-background") area = el;
+
+      // don't apply effect to focused url bar
+      if (this._options.includeUrlBar && el.id == 'urlbar-background' && window.gURLBar.focused) {
```

Comment:
```suggestion
      if (this._options.includeUrlBar && el.id == 'urlbar-background' && gURLBar.focused) {
```

gURLBar is a global variable so you don't need `window.`